tests/validate.test.js

```diff
@@ -434,6 +434,123 @@ describe('validate', () => {
+            'handlers/func3/module2': './handlers/func3/module2.js',
+            'handlers/module2/func3/module2': './handlers/module2/func3/module2.js'
+          },
+          output: {
+            path: 'output',
+          },
+        };
+
+        beforeEach(() => {
+          _.set(module.serverless, 'service.package.individually', 'true');
+        });
+
+        afterEach(() => {
+          _.unset(module.serverless, 'service.package.individually');
+        });
+
+        it('should enable multiCompile', () => {
+          module.serverless.service.custom.webpack = testConfig;
+          module.serverless.service.functions = testFunctionsConfig;
+          globSyncStub.callsFake(filename => [_.replace(filename, '*', 'js')]);
+
+          expect(module.multiCompile).to.be.undefined;
+          return expect(module.validate()).to.be.fulfilled
+          .then(() => {
+            expect(module.multiCompile).to.be.true;
+
+            return null;
+          });
+        });
+
+        it('should expose all functions details in entryFunctions property', () => {
+          module.serverless.service.custom.webpack = testConfig;
+          module.serverless.service.functions = testFunctionsConfig;
+          globSyncStub.callsFake(filename => [_.replace(filename, '*', 'js')]);
+          return expect(module.validate()).to.be.fulfilled
+          .then(() => {
+            expect(module.entryFunctions).to.deep.equal([
+              {
+                handlerFile: 'module1',
+                funcName: 'func1',
+                func: testFunctionsConfig.func1,
+                entry: { key: 'module1', value: './module1.js' }
+              },
+              {
+                handlerFile: 'module2',
+                funcName: 'func2',
+                func: testFunctionsConfig.func2,
+                entry: { key: 'module2', value: './module2.js' }
+              },
+              {
+                handlerFile: 'handlers/func3/module2',
+                funcName: 'func3',
+                func: testFunctionsConfig.func3,
+                entry: { key: 'handlers/func3/module2', value: './handlers/func3/module2.js' }
+              },
+              {
+                handlerFile: 'handlers/module2/func3/module2',
+                funcName: 'func4',
+                func: testFunctionsConfig.func4,
+                entry: { key: 'handlers/module2/func3/module2', value: './handlers/module2/func3/module2.js' }
+              }
+            ]);
+            return null;
+          });
+        });
+
+        it('should set webpackConfig output path for every functions', () => {
+          module.serverless.service.custom.webpack = testConfig;
+          module.serverless.service.functions = testFunctionsConfig;
+          globSyncStub.callsFake(filename => [_.replace(filename, '*', 'js')]);
+          return expect(module.validate()).to.be.fulfilled
+          .then(() => {
+            expect(module.webpackConfig.length).to.equal(4);
```

Comment:
Could be expressed as `expect(module.webpackConfig).to.have.lengthOf(4);` which will implicitly check for an array like structure.